iceoryx_examples/icedelivery/README.md

```diff
@@ -251,43 +260,21 @@ After the creation, the subscriber object subscribes to the offered data
 mySubscriber.subscribe();
 ```
 
-The next step is the instantiation and setup of the `WaitSet`
-```cpp
-iox::popo::WaitSet<> waitSet;
-waitSet.attachEvent(untypedSubscriber, iox::popo::SubscriberEvent::HAS_SAMPLES);
-waitSet.attachEvent(shutdownTrigger);
-```
-
-Here our subscriber object `untypedSubscriber` got attached to the WaitSet, as well as the `shutdownTrigger`.
-It will be used to stop the programm execution.
-
-Again in a while-loop we do the following: First wait till a `Trigger` has been sent to the `WaitSet` by calling
-```cpp
-auto triggerVector = waitSet.wait();
-```
-
-Then loop over all the trigger that got fired
+Again in a while-loop we do the following: First check for the `SubscriptionState`
```

Comment:
Please remove the Waitset related includes in line 232.

Reply:
Fixed